test/integration/Delegation.js

```diff
@@ -421,6 +425,8 @@ describe("Delegation", () => {
             startMinterTokenBalance.sub(BigNumber.from(700)).toString(),
             "wrong minter token balance after withdrawStake"
         )
+        await roundsManager.mineBlocks(newRoundsLength)
```

Comment:
It seems that the next test case starts with block progression + round initialization. Can we omit these lines and let the next test case handle setting up the test properly? If we update the global `roundLength` var then subsequent test cases would use the new roundLength value for handling block progression.

Reply:
Since this is an integration test it shares the same contract state across test cases, so either way it needs to be reset. I think resetting it to the original value right after that test makes more sense instead of resetting in next state.

Reply:
If the next test also calls mineBlocks() and initializeRound() immediately at the start of the test *and* we do the same here and the end of this test wouldn't we be initializing 2 rounds when we only need to initialize 1 round to setup the next test?

Reply:
I agree with you. I had added mineBlocks() based on my previous thought about finishing the round first before updating the roundLength. But based on comments here https://github.com/livepeer/internal-project-tracking/issues/400#issuecomment-1251065972 
I am now convinced to remove this